packages/EDA/src/EDAtools.ts

```diff
@@ -44,3 +44,19 @@ export async function computePLS(table: DG.DataFrame, features: DG.ColumnList, p
 
   return _output;
 } 
+
+// Analysis of Variance (ANOVA)
+export async function computeANOVA(table: DG.DataFrame, columns: DG.ColumnList): Promise<number> 
+{
+  checkColumns(columns);
+
+  let _output: any;
```

Comment:
Specify the actual type instead of "any".

Name variable in camelCase, why the underscore?

Reply:
Almost all parts of this code are generated automatically by C/C++-to-TS export script.
Now, all changes should be introduced manually. But further, I'll fix them in the export script.

Reply:
In other words, this is my fault.